### PR TITLE
Add customization docs

### PR DESCRIPTION
**Pull request by daileytj**
Fixes 4529.

#### Changes proposed in this Pull Request:

- Customization Docs
- error management docs
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

- Clone this branch and review the customization and error management docs. The routing and internationalization docs are actively being worked on now so those will be covered by those sections more specifically.

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by JeffGreiner-eaton** [comment]
I can't comment in the file for this but the error management needs updated in this section. (true or false)

image

**Comment by daileytj** [comment]
> I can't comment in the file for this but the error management needs updated in this section. (true or false)
> 
> image

What does it need to be updated to say?

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good to me

File: login-workflow/docs/customization.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+## Customizing Registration Workflow
+
+### Customizing the Screen Order
+You may want to customize the order of your screens within the workflow. In order to do this you will need to import the screens you want to use and pass them as children to the `RegistrationWorkflow` component. The screens will render in the order they are passed in.
+
+Our default implementation looks like this:
+  
+```jsx
+<RegistrationWorkflow initialScreenIndex={0}>
```
Comment (surajeaton) [inline login-workflow/docs/customization.md:12]: In default workflow, we don't need to pass initialScreenIndex={0}, as it is set to 0 by default.

File: login-workflow/docs/customization.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    <CreateAccountScreen />    // screen1
+    <MyCustomScreen />         // screen2
+    <CreatePasswordScreen />   // screen3
+    <AccountDetailsScreen />   // screen4
+</RegistrationWorkflow>
+```
+
+<!-- ### Customizing the Registration Success Screen -->
+<!-- @TODO... or not to do. This is just a prop on the screens it is used for. Do we need an in depth example or do the API docs cover it? -->
```
Comment (surajeaton) [inline login-workflow/docs/customization.md:36]: I believe it should be a part of RegistrationWorkflow component doc. May be add a line about that and a link to registration workflow component doc. https://github.com/etn-ccis/blui-react-workflows/blob/6ebfced227886744333deac6c4bd4c29f0ecfa3e/login-workflow/docs/components/registration-workflow.md?plain=1#L1

We need to tell users:
- How to add custom success screen to workflow.
- What is the default success screen look (may need to add link to success screen doc). We may need to guide success screen needs organization name and email in the success message, if user is not using organization name and email address in their workflow, they must need to pass custom success screen.
- We shouldn't pass success screen as child.
- An example for custom success screen.

**Pull request merged**